mvpa2/clfs/warehouse.py

```diff
@@ -47,7 +47,7 @@
         'libsvm', 'sg', 'meta', 'retrainable', 'gpr',
         'notrain2predict', 'ridge', 'blr', 'gnpp', 'enet', 'glmnet',
         'gnb', 'plr', 'rpy2', 'swig', 'skl', 'lda', 'qda',
-        'random-forest', 'extra-trees', 'random']
+        'random-forest', 'extra-trees', 'random', 'oneclass', 'oneclass-binary']
```

Comment:
sorry for a delay with the review and thanks for the contribution!! would you mind just adding 1 little clarification here in the comment by each "oneclass" and "...-binary" of what is the difference there ;)